inlong-sort/sort-connectors/elasticsearch-7/src/main/java/org/apache/inlong/sort/elasticsearch7/table/Elasticsearch7Configuration.java

```diff
@@ -68,4 +69,38 @@ private static HttpHost validateAndParseHostsString(String host) {
                     e);
         }
     }
+
+    public List<HttpHost> getHosts() {
+        return config.get(HOSTS_OPTION).stream()
```

Comment:
npe here

Reply:
It will not cause npe, because the 'HOST_OPTION' is a required option and valid before.
<img width="941" alt="image" src="https://user-images.githubusercontent.com/44659300/224650832-0403b646-6e5a-49fa-9ded-d059fabb93e1.png">